### DISCUSSION TITLE
how to configure the backroll

### DISCUSSION THREAD

**Discussion by nsaikrishna15** [DIMSI-IS/BackROLL#16] [Q&A]
Hi team, I would like to know how to configure the backroll  for the cloudstack? 
I have configured the backroll on a vm and the ui is not showing up. i am using the keycloak openid provider. can we have detailed setup my team and i are trying to set it up but facing issues bringing up the ui.

**Comment by nsaikrishna15** [comment]
i am getting a black page with latest image

**Reply by JoffreyLuang** [reply]
Hi,

How is your keycloak setup? Is it in docker in the same VM? 
We had unstable result with containerized keycloak with Backroll.

**Reply by nsaikrishna15** [reply]
its containerised. so we are facing too many issues and also this plugin is not available in the cloudstack-management so we decided to go with using cloudstack api to take vm snapshots and do it through automation scripts and jenkins. Thanks for the reply.
